polaris.shopify.com/src/components/Markdown/components/DirectiveCard/index.tsx

```diff
@@ -0,0 +1,65 @@
+import {Icon, Bleed} from '@shopify/polaris';
+import {
+  CancelSmallMinor,
+  FlagMajor,
+  RiskMajor,
+  TickSmallMinor,
+} from '@shopify/polaris-icons';
+
+import {Box} from '../../../Box';
+import {Card} from '../../../Card';
+import {Stack, Row} from '../../../Stack';
+import ImageThumbnail from '../../../ThumbnailPreview';
+import styles from './styles.module.scss';
+
+export enum DirectiveStatusName {
```

Comment:
Not sure how I feel about ts enum as an API, but using it here for consistency since we also use this pattern in `StatusBadge`.